src/include/migraphx/op/iff.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_IFF_HPP
+#define MIGRAPHX_GUARD_OPERATORS_IFF_HPP
+
+#include <array>
+#include <migraphx/check_shapes.hpp>
+#include <migraphx/argument.hpp>
+#include <migraphx/functional.hpp>
+#include <migraphx/config.hpp>
+#include <cmath>
+#include <utility>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace op {
+
+struct iff
+{
+    std::string then_sub_graph;
+    std::string else_sub_graph;
```

Comment:
Why are the subgraphs strings? I dont think the operator would store the subgraph since that is already stored in the instruction.

Reply:
removed